src/content/guides/creating-clusters-via-crs-on-aws/index.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Creating tenant clusters on AWS via Control Plane Kubernetes API"
+description = "This guide will walk you through the process of tenant cluster creation via Control Plane Kubernetes."
+date = "2020-04-01"
+type = "page"
+weight = 100
+tags = ["tutorial"]
++++
+# Creating tenant clusters directly via Control Plane Kubernetes API
+
+Starting from version 10.0.0, Giant Swarm introduced [node pools](https://docs.giantswarm.io/basics/nodepools/) feature for AWS platform.
+Alongside node pools support, a new API version for cluster management was released. 
+Therefore, [v5](https://docs.giantswarm.io/api/#operation/addClusterV5) API is used now within happa and [gsctl](https://docs.giantswarm.io/reference/gsctl/create-cluster/).
+
+As for now, Giant Swarm is moving towards replacing its own API with the direct cluster management via [Control Plane Kubernetes API](https://docs.giantswarm.io/basics/aws-architecture/#giant-swarm-control-plane).
+Following this movement strategy, Giant Swarm API is going to be deprecated in the near feature. You can find related roadmap issue [here](https://github.com/giantswarm/roadmap/issues/90).
+
+## How does cluster creation works now?
+
+All the tenant clusters, created with release version 10.x.x+, are managed as [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in Control Plane Kubernetes.
+That means, when you're creating a new cluster via [v5](https://docs.giantswarm.io/api/#operation/addClusterV5) API, what *API* service does, it uses [Cluster API](https://github.com/kubernetes-sigs/cluster-api) to create tenant Kubernetes control-plane CR and optional node pools CRs.
```

Comment:
Same here. The API always created the necessary CRs for the operators to create the infrastructure.